sktime/classification/feature_based/_catch22_classifier.py

```diff
@@ -135,32 +135,34 @@ def _fit(self, X, y):
         return self
 
     def _predict(self, X):
-        """Predict class values of n_instances in X.
+        """Predict class values of n instances in X.
 
         Parameters
         ----------
-        X : pd.DataFrame of shape (n_instances, n_dims)
+        X : 3D np.array of shape = [n_instances, n_dimensions, series_length]
+            The data to make predictions for.
 
         Returns
         -------
-        preds : np.ndarray of shape (n, 1)
-            Predicted class.
+        y : array-like, shape = [n_instances]
```

Comment:
Same here array-like isnt a type

Reply:
so we have used array-like historically. I think it needs to be np.array at the moment, I dont think conversion to numpy from df is actually implemented (I'm putting it in on another branch)